docs/src/snap/explanation/ingress.md

```diff
@@ -0,0 +1,79 @@
+# Ingress
+
+In Kubernetes, understanding how inbound traffic is managed inside of your
+cluster can be complex.
+While there is abundant documentation, this explanation provides you with the essentials
+to successfully manage your Canonical Kubernetes cluster.
+
+## Kubernetes Pods and Services
+
+In Kubernetes, the smallest unit is a pod, which encapsulates application containers.
+Since pods are ephemeral and their IP addresses change when destroyed and restarted,
+they are exposed through services.
+Services offer a stable network interface by providing discoverable names and
+load balancing functionality for managing a set of pods.
+For further details on Kubernetes Services,
+refer to the upstream documentation: [Kubernetes Service][Service].
+
+## Ingress
+
+[Ingress][Ingress K8s] is a Kubernetes Resource that manages
+external access by handling both HTTP and HTTPS traffic to services within your cluster.
+Traffic routed through the Ingress is directed to a service,
+which in turn forwards it to the relevant pod
+running the desired application within a container.
+
+The Ingress Resource lets you define rules on how traffic should get handled.
+Learn more about these rules and how to implement them here:
```

Comment:
probably better to say "Refer to the [Kubernetes documentation on Ingress rules] for up to date information on the available rules and their implementation." or something like that.